### DISCUSSION TITLE
RxRx1 Data Augmentation

### DISCUSSION THREAD

**Discussion by niki-amini-naieni** [p-lambda/wilds#118] [Q&A] [answered]
Hello, 

It seems as though RxRx1 already includes data augmentation for the training data for all baseline methods as shown below
<img width="516" alt="image" src="https://user-images.githubusercontent.com/61301874/161976414-7abc4658-00d8-407f-b89f-4996921f33d9.png">
(https://github.com/p-lambda/wilds/blob/main/examples/transforms.py)

Could you please confirm that this is the case and provide the reasoning for this decision? Why was this done for RxRx1 and not for some other datasets?

Thank you!

**Comment by kohpangwei** [comment, marked as answer]
Yes, it does include data augmentation. For each dataset, we picked some default augmentations based on prior work, and this augmentation was what was used in our reference implementation for RxRx1.

**Reply by niki-amini-naieni** [reply]
Great, thanks for the confirmation!
